phpstan-baseline.neon

```diff
@@ -10,6 +10,11 @@ parameters:
 			count: 2
 			path: src/CssInliner.php
 
+		-
+			message: "#^Casting to int something that's already int\\<0, max\\>\\.$#"
```

Comment:
Does this mean we'd need to do something like `is_int($var) ? $var : (int)$var`.  I don't like that.  But we can review that section of code and see how it could be improved.

Reply:
No, more like this: #1290